src/org/zaproxy/zap/extension/ascanrulesBeta/SQLInjectionHypersonic.java

```diff
@@ -214,6 +219,9 @@ public void init() {
 			doTimeBased=true; doTimeMaxRequests=100;
 			doUnionBased=true; doUnionMaxRequests=100;
 		}
+		// Read the sleep value from the configs - note this is in milliseconds
+		this.sleep = this.getConfig().getInt("rules.common.sleep", 5) * 1000;
+		if ( this.debugEnabled ) log.debug("Sleep set to " + sleep + " seconds");
```

Comment:
Should be "milliseconds" instead of "seconds"? (or converted to seconds)
The if could have curly brackets.